article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -1,91 +1,9 @@
 package services.dotcomponents
 
-import controllers.ArticlePage
+import dotcomrendering.DotcomRenderingUtils._
```

Comment:
why 'Utils'?

Reply:
No real reason. I could easily rename it. Actually let me rename it :)

Reply:
Done!